announcement/demo/announcement_tag_demo.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="announcement_tag_1" model="announcement.tag">
+        <field name="name">Tag 1</field>
```

Comment:
Try to put significant tags, not numerated tags. This can showcase one of the uses of such tags.